dist/main.css

```diff
@@ -0,0 +1,34 @@
+.title {
+  font-size: 3rem;
+  font-weight: 600;
+  color: #2dcbfd;
+}
+
+.title span {
+  color: #fdbb2d;
+}
+
+.figures-container {
+  background-color: #4158d0;
+  background-image: linear-gradient(
+    43deg,
+    #4158d0 0%,
+    #c850c0 46%,
+    #ffcc70 100%
+  );
+  width: fit-content;
+}
+
+.switch {
+  font-size: 2rem;
```

Comment:
### OPTIONAL 

- [ ] It would be better if we implement the CSS DRY principle, there are a couple of repeating rules; for example `font-size: 2rem`, `font-size:2.3rem` `font-size: 1.5rem` and a few other rules have been repeated